view/adminhtml/ui_component/category_form.xml

```diff
@@ -13,7 +13,7 @@
                 <item name="config" xsi:type="array">
                     <item name="dataType" xsi:type="string">string</item>
                     <item name="source" xsi:type="string">category</item>
-                    <item name="label" xsi:type="string" translate="true">PureClarity Secondary Image</item>
+                    <item name="label" xsi:type="string" translate="true">Override Image</item>
```

Comment:
Should we call this PureClarity Override Image?

Reply:
I didn't think was needed myself as you already have to drop the PureClarity tab/section to view, seemed superfluous? Can update if you want though.

Reply:
Ok that's fine then